Makefile

```diff
@@ -45,3 +45,7 @@ else
 include tools/Makefile.unix
 endif
 endif
+
+.PHONY: execname
+execname:
+	@echo $(BIN)
```

Comment:
why not copy the binary directly? it's strange to print the binary name in Makefile

Reply:
I want a target that allows other scripting to use the files (like out CI system). I don't want the makefile to actually do anything with them, right now there is no way to get this info. We could generate an manifest file if you prefer.

Reply:
Manifest file is now generated by the build that can be used.

Reply:
You don't need this anymore, right?

Reply:
@btashton let's revert this change?

Reply:
Must have slipped in when I was resolving merge conflicts.  I have removed and rebased to fix the other conflict.